TheJumpCloud/JumpCloud-Serverless
# Singleline JSON Format + Added Services

## Issues
* [SA-3723](https://jumpcloud.atlassian.net/browse/SA-3723) - SA-3723-Flatten-Json-Option

## What does this solve?
This change adds an option to choose json formatting between MultiLine and SingleLine. Also added new services password_manager, software, and object_storage
## Is there anything particularly tricky?
n/a

## How should this be tested?
Deploy two instances, one with  `--JsonFormat=SingleLine` and one with` --JsonFormat=MultiLine` json format. Also, make sure the newly added services are covered. Validate that both data stored in S3 is correct.


## Screenshots


[SA-3723]: https://jumpcloud.atlassian.net/browse/SA-3723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
